tests/integration/test_core.py

```diff
@@ -353,3 +373,69 @@ async def test_openstack_object_storage_plugin(
             assert (
                 requests.get(url).content == image
             ), "image downloaded from WordPress should match the image uploaded"
+
+
+@pytest.mark.asyncio
+async def test_openstack_akismet_plugin(
+    ops_test: pytest_operator.plugin.OpsTest,
+    application_name,
+    default_admin_password,
+    unit_ip_list,
+    akismet_api_key,
+):
+    """
+    arrange: after WordPress charm has been deployed, db relation established
+    act: update charm configuration for Akismet plugin
+    assert: Akismet plugin should be activated and spam detection function should be working
+    """
+    application = ops_test.model.applications[application_name]
+    await application.set_config({"wp_plugin_akismet_key": akismet_api_key})
```

Comment:
Do you think the configuration should be reset after the test completes so as not to impact other integration tests? If so, do you think this should be in a fixture to guarantee that the configuration change is reverted?

Reply:
There's a fixture for config and revert config using fixture parameter. But this test case requires `akismet_api_key` which is a fixture, and you can't pass a fixture to a fixture parameter.